tools/cli/lib/installcmd.c

```diff
@@ -217,20 +217,12 @@ TDNFCliAlterCommand(
 
     if(pSolvedPkgInfo->nNeedAction)
     {
-        int32_t opt = 0;
+        int nAnswer = 0;
 
-        if(!pCmdArgs->nAssumeYes && !pCmdArgs->nAssumeNo)
-        {
-            printf("Is this ok [y/N]: ");
-
-            opt = getchar();
-            if (tolower(opt) != 'y' && tolower(opt) != 'n')
-            {
-                printf("Invalid input\n");
-            }
-        }
+        dwError = TDNFYesOrNo(pCmdArgs, "Is this ok [y/N]", &nAnswer);
```

Comment:
Please use the exact prompt string as it was before, there may be some test case which might rely on it.